src/components/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {Box,Button} from '@mui/material'
+import { Note } from './customComponents/Note';
+import { Navbar } from './customComponents/Navbar';
+import { SearchBar } from './customComponents/SearchBar';
+
+export const Home = () => {
+    return <>
+    <div className="wrapper">
+    <aside>
+      <Box mb={4} sx={{ width: '100%', minWidth: 300, bgcolor: 'background.paper'}}>
```

Comment:
If there are more than 3 styles adding to an element. Consider styling them in a seperate stylesheet

Reply:
Sure. Have considered doing this. Separated out the styles in a separate file.